lib/ui/screen/main.dart

```diff
@@ -60,22 +66,29 @@ class _MyHomePageState extends State<MyHomePage> {
   Future refreshAllAlerts() async {
     setState(() => isLoading = true);
 
-    allAlert = await AlarmDatabase.instance.readAllAlerts();
+    allAlerts = await AlarmDatabase.instance.readAllAlerts();
+    print(_currentFilter);
```

Comment:
avoid using print in production code

Reply:
Ack. This is meant to be a sanity check of the filter value (since now alerts only have the `pending` status). Will change it to debug log